engine/src/test/java/org/destinationsol/testingUtilities/InitializationUtilities.java

```diff
@@ -33,7 +34,7 @@ public static void init() {
         }
         initialized = true;
         DebugOptions.DEV_ROOT_PATH = "engine/src/main/resources/";
-        final LwjglApplication application = new LwjglApplication(new SolApplication());
+        final LwjglApplication application = new LwjglApplication(new SolApplication(new ModuleManager()));
         try {
             Thread.sleep(10000L); // Magic happens here. If too much tests fail for you, increase this number.
```

Comment:
About the "Magic" - it was just a quick and dirty way to assure modules are loaded and game is initialized before running the tests - could you check if this is still needed and/or can be easily replaced? (Maybe moving the module initialization fixed this)